presets/consumer/avformat/Screencast-H264

```diff
@@ -0,0 +1,23 @@
+pix_fmt=yuv420p
+vcodec=libx264
+vprofile=high
+level=51
```

Comment:
level should be left out because it depends mainly on the resolution and frame rate.

Reply:
Removed.